docs/index.asciidoc

```diff
@@ -5,12 +5,13 @@
 include::{asciidoc-dir}/../../shared/versions/stack/current.asciidoc[]
 include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
 
-:ecs_github_repo_link: https://github.com/elastic/ecs/blob/master
+:ecs_github_repo_link: https://github.com/elastic/ecs/blob/{source_branch}
+:ecs_generated_version: 2.0.0-dev
 
 [[ecs-reference]]
 == Overview
 
-This is the documentation of ECS version 2.0.0-dev.
+This is the documentation of ECS version 2.0.0-dev
```

Comment:
```suggestion
This is the documentation of ECS version 2.0.0-dev.
```

The period is getting left off in the new template.

Reply:
Agree, period restored